### PR TITLE
Feature/acceleration limiter

### PR DESCRIPTION
**Pull request by rolfvdhulst**
updates the acceleration limiter to be kind to our motors.

### CODE REVIEW

File: roboteam_ai/src/control/ControlUtils.cpp [hunk 1]
```diff
@@ -235,7 +235,22 @@ double ControlUtils::calculateMaxAcceleration(const Vector2 &vel, double angle)
     double a = abs(toVectorDiff.x);
     return Constants::MAX_ACC_UPPER() * (a) + Constants::MAX_ACC_LOWER() * (1-a);
 }
-
+Vector2 ControlUtils::accelerationLimiterNew(const Vector2 &vel,double robotAngle, const Vector2 &prevVel) {
+    // calculate max
+    Vector2 diff=vel-prevVel;
+    Angle angleDiff=diff.angle()-robotAngle;
+    Vector2 toVectorDiff=angleDiff.toVector2();
+    double a=abs(toVectorDiff.x);
```
Comment (simenbootsma) [inline roboteam_ai/src/control/ControlUtils.cpp]: Why use toVectorDiff, isn't `a=abs(cos(angleDiff))` sufficient?

Comment (rolfvdhulst) [reply roboteam_ai/src/control/ControlUtils.cpp]: This was exactly the same code as you used previously. It's functionally the same, so I don't mind changing it.

File: roboteam_ai/src/control/positionControllers/PosController.cpp [hunk 2]
```diff
@@ -35,12 +35,17 @@ PosVelAngle PosController::controlWithPID(const RobotPtr &robot, PosVelAngle tar
     pidCommand.vel = calculatePIDs(robot, target);
 
     // acceleration limiter
-    double maxAcc = control::ControlUtils::calculateMaxAcceleration(pidCommand.vel, pidCommand.angle);
-    if (prevVel == 0.0) prevVel = robot->vel.length();
-    pidCommand.vel = control::ControlUtils::accelerationLimiter(pidCommand.vel, maxAcc, prevVel);
+    if (prevVel == Vector2(0.0,0.0)) {
+        prevVel = robot->vel;
+    }
+    //OLD
```
Comment (mrlukasbos) [inline roboteam_ai/src/control/positionControllers/PosController.cpp]: we have git so you don't need this

Comment (rolfvdhulst) [reply roboteam_ai/src/control/positionControllers/PosController.cpp]: this was for testing purposes so recompiling does not take ages. We were still testing this

**Pull request closed without merging**